community/commercial-support.html.in

```diff
@@ -126,40 +153,13 @@ auditing.
     </li>
 
     <li>
-      <a href="http://blackriversoft.com/">
-        <div>
-          <img alt="Black River Software" src="/community/commercial-support-logos/black-river-software.png" />
-        </div>
-        <h2>Black River Software</h2>
-        <ul><li>Ohio, USA</li></ul>
-        Black River Software offers custom software development, software architecture
-consulting, and build and deployment engineering services.
-
-      </a>
-    </li>
-
-    <li>
-      <a href="https://helsinki-systems.de">
-        <div>
-          <img alt="Helsinki Systems" src="/community/commercial-support-logos/helsinki-systems.png" />
-        </div>
-        <h2>Helsinki Systems</h2>
-        <ul><li>Stuttgart, Germany</li></ul>
-        Your partner for hosting, networks and IT solutions running on NixOS. We have
-multiple years of NixOS and decades of Linux experience to aid your NixOS
-integration and administration.
-
-      </a>
-    </li>
-
-    <li>
-      <a href="https://www.enlambda.com/">
+      <a href="https://obsidian.systems">
         <div>
-          <img alt="Enlambda" src="/community/commercial-support-logos/enlambda.png" />
+          <img alt="Obsidian Systems" src="/community/commercial-support-logos/obsidian-systems.svg" />
         </div>
-        <h2>Enlambda</h2>
-        <ul><li>Remote</li><li>Estonia</li></ul>
-        Turning operational issues into development tasks using Nix since 2013.
+        <h2>Obsidian Systems</h2>
+        <ul><li>New York, USA</li></ul>
+        Software consultancy specializing in Haskell, Nix, curiosity, and innovation.
 
       </a>
     </li>
```

Comment:
Remove these changes from your commit; it's being updated by an action regularly.